utilities/cli-runner/src/main/kotlin/org/jetbrains/kotlin/cli/utilities/main.kt

```diff
@@ -40,5 +42,17 @@ private fun mainImpl(args: Array<String>, konancMain: (Array<String>) -> Unit) {
 
 fun main(args: Array<String>) = mainImpl(args, ::konancMain)
 
-fun daemonMain(args: Array<String>) = mainImpl(args, ::konancMainForGradle)
+private fun setupClangEnv() {
+    setEnv("LIBCLANG_NOTHREADS", "1")
```

Comment:
Is `LIBCLANG_NOTHREADS` actually required?

Reply:
Seems that it's not, will remove.